### PR TITLE
SIANXKE-178: enhanced configuration options

### PR DESCRIPTION
**Pull request by anx-abruckner**
* additional configuration options to add/remove partial logs (request + body/headers, response)
* enhanced controll via settings (manage views globally) and decorator (manage single views)
* README update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 django-request-logging
 ==========================
 
-Plug django-request-logging into your Django project and you will have intuitive and color coded request/response payload logging, for both web requests and API requests. Supports Django 1.8+.
+Plug anx-django-request-logging (a fork of (Rhumbix django-request-logging)[https://github.com/Rhumbix/django-request-logging/]) into your Django project and you will have intuitive and color coded request/response payload logging, for both web requests and API requests. Supports Django 1.8+.
```
Comment (nezhar) [inline README.md]: Please remove the `anx-` prefix and the mention that this is a fork, this information is already available with GitHub:
Selection_182

File: README.md [hunk 2]
```diff
@@ -31,11 +31,14 @@ LOGGING = {
         'console': {
             'class': 'logging.StreamHandler',
         },
+        'database': {
+            'class': 'logging.'
```
Comment (nezhar) [inline README.md]: This should be removed, it adds no value to the docs. Maybe we can refer to https://docs.djangoproject.com/en/4.0/topics/logging/

File: README.md [hunk 3]
```diff
@@ -31,11 +31,14 @@ LOGGING = {
...
+        },
     },
     'loggers': {
         'django.request': {
-            'handlers': ['console'],
-            'level': 'DEBUG',  # change debug level as appropiate
+            'handlers': ['console', 'database'],  # change handlers as appropriate
```
Comment (nezhar) [inline README.md]: Same as above, we only want to maintain a minimal example here

File: setup.py [hunk 4]
```diff
@@ -39,14 +39,14 @@ def get_version(package):
 
 
 setup(
```
Comment (nezhar) [inline setup.py]: This file should not be changed.

File: request_logging/__init__.py [hunk 5]
```diff
@@ -1 +1 @@
-__version__ = "0.7.4"
+__version__ = "1.0.0"
```
Comment (nezhar) [inline request_logging/__init__.py]: This file should not be changed.

**Review by nezhar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anx-abruckner) [reply README.md]: Removed.

File: README.md (hunk 2 above)
Comment (anx-abruckner) [reply README.md]: Removed.

File: README.md (hunk 3 above)
Comment (anx-abruckner) [reply README.md]: Removed.

File: setup.py (hunk 4 above)
Comment (anx-abruckner) [reply setup.py]: Reverted.

File: request_logging/__init__.py (hunk 5 above)
Comment (anx-abruckner) [reply request_logging/__init__.py]: Reverted.

File: README.md [hunk 6]
```diff
@@ -73,13 +106,23 @@ By default, HTTP status codes between 400 - 499 are logged at ERROR level.  You
 If you set `REQUEST_LOGGING_HTTP_4XX_LOG_LEVEL=logging.INFO` they will be logged the same as normal requests.
 ### REQUEST_LOGGING_SENSITIVE_HEADERS
 The value of the headers defined in this settings will be replaced with `'*****'` to hide the sensitive information while logging. By default it is set as `REQUEST_LOGGING_SENSITIVE_HEADERS = ["HTTP_AUTHORIZATION", "HTTP_PROXY_AUTHORIZATION"]`
+### DJANGO_REQUEST_LOGGING_LOGGER_NAME
+Name of the logger that is used to log django.request occurrances with the new LoggingMiddleware. Defaults to "django.request".
+### DJANGO_REQUEST_LOGGING_NO_LOGGING_DEFAULT_VALUE
+Global default to activate/deactivate logging of all views. Can be overruled for each individual view by using the @no_logging decator's "value" parameter.
+### DJANGO_REQUEST_LOGGING_LOG_HEADERS_DEFAULT_VALUE = False
```
Comment (nezhar) [inline README.md]: Please adapt the default value to `True`. Also for `DJANGO_REQUEST_LOGGING_LOG_BODY_DEFAULT_VALUE` and `DJANGO_REQUEST_LOGGING_LOG_RESPONSE_DEFAULT_VALUE`

Comment (anx-abruckner) [reply README.md]: Already is ``True`` in the code, was just wrong in the README. Fixed that =)

**Review by nezhar — APPROVED** [review]

**Pull request merged**